### ISSUE TITLE
Feature: Show consistency status of PI/sensitivity analysis (e.g. by green/red traffic lights)

### ISSUE THREAD

**Issue by Christoph27** [Open-Systems-Pharmacology/PK-Sim#2467]
If the user runs a parameter identification and subsequently changes in the corresponding simulation a parameter (which was not used in the parameter identification), the PI result is not anymore consistent with the simulation. The user can currently not see this. It would be good to have e.g. a traffic light system with red and green symbols (similar to that for building blocks) to indicate the consistency status of PI runs with simulation(s). Same for sensitivity analysis runs.

**Comment by PavelBal** [comment]
Well this is only the case when changing the simulation while the PI is running. I think it cannot be expected that the changes are applied in real time.

**Comment by Christoph27** [comment]
No, just an example: you have a model with a certain value for the glomerular filtration rate and a certain value of the specific CYP3A4 CL. You run a PI for the specific CYP3A4 CL only and get an optimized value for that. Then - maybe weeks later - you change the value for the glomerular filtration rate in the simulation. After that, the PI is not consistent any mor with the simulation since re-running the PI would give a different value for the specific CYP3A4 CL.

**Comment by PavelBal** [comment]
True.
